CollAction/Controllers/ProjectsController.cs

```diff
@@ -438,7 +438,8 @@ public ViewResult StartInfo()
             Expression<Func<Project, bool>> statusExpression;
             switch (statusId)
             {
-                case (int)ProjectExternalStatus.Open: statusExpression = (p => p.Status == ProjectStatus.Running && p.Start <= DateTime.UtcNow && p.End >= DateTime.UtcNow); break;
+                case (int)ProjectExternalStatus.OpenandClosed:
```

Comment:
If we're outputting both the open and the closed projects by default, I'd find it logical to display the open projects above the closed projects. Not sure what the sort order is now, by if it's sorted by id it won't always be sufficient.

Maybe ask ron what exactly he wants?